index.js

```diff
@@ -4,6 +4,9 @@ import getRawBody from 'raw-body'
 import assert from 'http-assert'
 import { validate } from './lib/validate.js'
 
+// Approximate length of each SPARK/Meridian round in milliseconds
+const ROUND_LENGTH_IN_MS = 30 * 60_000 /* ms */
```

Comment:
as discussed, this can be queried from the contract:

```suggestion
const ROUND_LENGTH_IN_MS = (await ie.nextRoundLength()) * 30_000 /* block time ms */
```

Reply:
> > Do you think it's worth the effort to remove this constant and query the round length whenever we detect the start of a new MERidian round?
Considering how infrequently we are going to change this constant, maybe it's fine to keep this hard-coded for now.
>
> We could either query at the start of each round, or at process start? It should be fine to query at the start of each round, since the call is free and should be reasonably fast.

I am not very comfortable querying this only once at the process start because contract admins can dynamically change the value.

I am going to query this whenever a new round starts.

Reply:
See [1cbf01d](https://github.com/filecoin-station/spark-api/pull/80/commits/1cbf01d7a1aa92b9d608848849b3f7a74347a65a)